charts/gsp-cluster/templates/00-aws-auth/aws-auth.yaml

```diff
@@ -18,30 +18,21 @@ data:
       groups:
         - system:masters
   {{- end }}
-  {{- range .Values.sreRoleARNs }}
-    - rolearn: {{ . | quote }}
-      username: sre
-      groups:
-        - sre
-  {{- end }}
   {{- range .Values.users }}
+  {{- $user := . }}
     - rolearn: {{ .roleARN | quote }}
       username: {{ .name }}
       groups:
-      {{- range .roles }}
-        - {{ .account }}-{{ .role }}
-      {{- end }}
-  {{- end }}
-  mapUsers: |
-  {{- range .Values.adminUserARNs }}
-    - userarn: {{ . | quote }}
-      username: admin
-      groups:
+        - aws-iam:authenticated-users
+        {{- if has .name $.Values.clusterAdmins }}
         - system:masters
-  {{- end }}
-  {{- range .Values.sreUserARNs }}
-    - userarn: {{ . | quote }}
-      username: sre
-      groups:
-        - sre
+        {{- end }}
+        {{- range $.Values.namespaces }}
+        {{- $namespace := . }}
```

Comment:
instead of this and the range below, how about:
```
{{- if has $user.name .operators }}
- {{ .name }}-operators
{{- end }}
```

Reply:
Nice! Updated.